### PR TITLE
Review candidate decision views

### PR DESCRIPTION
**Pull request by paulrobertlloyd**
## Context

Detailed review of decision views in the candidate interface.

## Changes proposed in this pull request

* Removes deprecated accept/decline via support view - no longer used since early MVP days 
* Cleans up decision views, ensuring consistent layout across different pages (i.e. heading being with 2/3 grid column)
* Corrects the repeated use of `.govuk-heading-xl` class for page headings
* Localises all decision page titles
* Localises all decision form labels and text
* Consistent naming for withdrawal feedback (aligning keywords around `withdrawal_feedback`)
* Consistent naming for accept and decline decisions (aligning keywords around `accept_offer` and `decline_offer`)
* Localise withdrawal feedback success flash message

## Guidance to review

Have moved most localisations for these pages into a new file called `decisions.yml` in the locales folder - might not be quite the right place, but this ensures all related strings are clearly grouped together.

## Things to check

- [ ] This code does not rely on migrations in the same Pull Request
- [ ] If this code includes a migration adding or changing columns, it also backfills existing records for consistency
- [ ] API release notes have been updated if necessary
- [ ] New environment variables have been [added to the Azure config](https://github.com/DFE-Digital/apply-for-teacher-training#azure-hosting-devops-pipeline)

### CODE REVIEW

**Comment by malcolmbaig** [comment]
Small typo in commit message https://github.com/DFE-Digital/apply-for-teacher-training/pull/3462/commits/11d09dd376e8dfc42467d026a6ecf79c475bfe26. 

Test failure is a flaky spec. I've raised a fix but triggering another test run should get this passing (:finger)

**Review by malcolmbaig — APPROVED** [review]

File: app/views/candidate_interface/decisions/offer.html.erb [hunk 1]
```diff
@@ -1,22 +1,20 @@
...
-  </div>
 
-  <%= f.govuk_error_summary %>
+      <%= f.govuk_error_summary %>
 
-  <h1 class="govuk-heading-xl govuk-heading-xl">
-    <span class="govuk-caption-xl"><%= @application_choice.offered_course.provider.name %> - <%= @application_choice.offered_course.name %> (<%= @application_choice.offered_course.code %>)</span>
-    <%= t('page_titles.view_and_respond_to_offer') %>
-  </h1>
+      <h1 class="govuk-heading-xl govuk-heading-xl">
```
Comment (davidgisbey) [inline app/views/candidate_interface/decisions/offer.html.erb]: dupe

File: app/views/candidate_interface/decisions/withdraw.html.erb [hunk 2]
```diff
@@ -1,22 +1,18 @@
 <% content_for :title, t('page_titles.withdraw_course_choice') %>
 <% content_for :before_content, govuk_back_link_to %>
 
-<h1 class="govuk-heading-xl govuk-heading-xl">
-  Are you sure you want to withdraw this course choice?
-</h1>
-
 <div class="govuk-grid-row">
   <div class="govuk-grid-column-two-thirds">
+    <h1 class="govuk-heading-xl govuk-heading-xl">
```
Comment (davidgisbey) [inline app/views/candidate_interface/decisions/withdraw.html.erb]: and again

File: app/views/candidate_interface/decisions/accept.html.erb [hunk 3]
```diff
@@ -1,18 +1,16 @@
 <% content_for :title, 'Are you sure you want to accept this offer?' %>
 <% content_for :before_content, govuk_back_link_to %>
 
-<h1 class="govuk-heading-xl govuk-heading-xl">
-  Are you sure you want to accept this offer?
-</h1>
-
 <div class="govuk-grid-row">
   <div class="govuk-grid-column-two-thirds">
+    <h1 class="govuk-heading-xl govuk-heading-xl">
```
Comment (davidgisbey) [inline app/views/candidate_interface/decisions/accept.html.erb]: dupe

File: app/views/candidate_interface/decisions/decline.html.erb [hunk 4]
```diff
@@ -4,25 +4,17 @@
 <div class="govuk-grid-row">
   <div class="govuk-grid-column-two-thirds">
     <%= render CandidateInterface::UCASDowntimeComponent.new  %>
-  </div>
-</div>
 
-<h1 class="govuk-heading-xl govuk-heading-xl">
-  Are you sure you want to decline this offer?
-</h1>
+    <h1 class="govuk-heading-xl govuk-heading-xl">
```
Comment (davidgisbey) [inline app/views/candidate_interface/decisions/decline.html.erb]: image

**Review by davidgisbey — APPROVED** [review]
Thought i'd caught some dupes, but should have known to finish my review before commenting :shipit:

**Pull request merged**